### ISSUE TITLE
Use VS2015's project name for nuget packages

### ISSUE THREAD

**Issue by bastte** [aspnet/Home#703]
It seems that the nuget packages generated by checking the "Product outputs on build" box in VS are named only after the folder name.

Since there are a lot of restrictions around path lengths in Windows we don't want to put a fully qualified name on the folder (we've already ran into issues building because of this), but we'd like our packages to be named something like "Company.TopLevelProduct.Package". I tried setting the AssemblyName in the xproj but it doesn't seem to care.

Is there a way to accomplish that (other than renaming the nupkg afterwards) ?

Thanks!

**Comment by davidfowl** [comment]
Not at the moment but you can follow this issue: https://github.com/aspnet/dnx/issues/1559

**Comment by bastuc** [comment]
Awesome, I don't know how I didn't find this issue before.
Thanks for the link, very interesting comments in there, I'll follow that closely.

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
